Content/0.4.12 - Security Shields/securityShields.json

```
@@ -14,5 +14,12 @@
 	"securityShields_assigned_onlyHiddenEntities": "Hidden Accounts or Personas",
 	"securityShields_changeMain_title": "Change Default Shield",
 	"securityShields_changeMain_subtitle": "Choose which Security Shield will be pre-selected when you want to use one for an Account or Persona.",
-	"securityShields_createShieldButton": "Create New Security Shield"
+	"securityShields_createShieldButton": "Create New Security Shield",
+	"securityShields_applied_accountsAndPersonas": "This Security Shield is applied to these Accounts and Personas",
+	"securityShields_accounts_title": "Accounts",
+	"securityShields_personas_title": "Personas",
+	"securityShields_noAccounts_title": "No Accounts",
+	"securityShields_noPersonas_title": "No Personas",
+	"securityShields_hiddenAccountsOrPersonas_title": "Hidden Accounts or Personas",
+	"securityShields_editFactors_title": "Edit Factors"
```

seems pointless to have a new level for each if there is only title in each category.
```suggestion
	"securityShields_applied_accountsAndPersonas": "This Security Shield is applied to these Accounts and Personas",
	"securityShields_accounts": "Accounts",
	"securityShields_personas": "Personas",
	"securityShields_noAccounts": "No Accounts",
	"securityShields_noPersonas": "No Personas",
	"securityShields_hiddenAccountsOrPersonas": "Hidden Accounts or Personas",
	"securityShields_editFactors": "Edit Factors"
```